data/com.github.donadigo.appeditor.appdata.xml

```diff
@@ -38,6 +38,7 @@
           <li>French translation (dabou)</li>
           <li>Russian translation (Cammelan)</li>
           <li>Spanish translation (Juan Gómez Carrillo)</li>
+          <li>German translation (Alexander Eisele)</li>
```

Comment:
Remove this line, this is the changelog for previous 1.0.0 version, since you're merging to master, the next version doesn't exist yet, I'll make sure to include the new translations along with their authors in the that version's changelog when I release.

Reply:
Ok